### PR TITLE
OptionsLayer with some bindings for GDL... again -_-

### PR DESCRIPTION
**Pull request by Kolyah35**

### CODE REVIEW

File: bindings/2.200/Cocos2d.bro [hunk 1]
```diff
@@ -739,6 +739,68 @@ class cocos2d::CCSprite {
...
+	virtual void setCascadeColorEnabled(bool);
+	virtual void updateDisplayedColor(cocos2d::_ccColor3B const&);
+	virtual bool isCascadeOpacityEnabled();
+	virtual void setCascadeOpacityEnabled(bool);
+	virtual void updateDisplayedOpacity(unsigned char);
+}
+
+[[link(win, android)]]
+class cocos2d::CCApplication {
+	virtual void openURL(char const*);
```
Comment (matcool) [inline bindings/2.200/Cocos2d.bro:803]: nice merging

File: bindings/2.200/GeometryDash.bro [hunk 2]
```diff
@@ -5622,6 +5631,9 @@ class GJTransformControlDelegate {
 }
 
 
+[[link(android)]]
+class MultilineBitmapFont : cocos2d::CCSprite {}
```
Comment (matcool) [inline bindings/2.200/GeometryDash.bro:5635]: this is already defined on line 12733, remove it

File: bindings/2.200/GeometryDash.bro [hunk 3]
```diff
@@ -7740,6 +7752,78 @@ class GJEffectManager : cocos2d::CCNode {
 	virtual bool init() = win 0x1d6f50;
 }
 
+[[link(android)]]
+class OptionsLayer : GJDropDownLayer, FLAlertLayerProtocol {
```
Comment (matcool) [inline bindings/2.200/GeometryDash.bro:7756]: this was already on line 8728, just add the addresses there please.

File: bindings/2.200/GeometryDash.bro [hunk 4]
```diff
@@ -7740,6 +7752,78 @@ class GJEffectManager : cocos2d::CCNode {
...
+	void sfxSliderChanged(cocos2d::CCObject*) = win 0x2ac0b0;
+	TodoReturn createToggleButton(gd::string, cocos2d::SEL_MenuHandler, bool, cocos2d::CCMenu*, cocos2d::CCPoint); // idk lol
+	void musicSliderChanged(cocos2d::CCObject*) = win 0x2abfc0;
+	void exitLayer() = win 0x2ab110;
+
+	virtual void customSetup() = win 0x2ab510;
+	virtual void layerHidden() = win 0x2ac660;
+}
+
+class GJAccountDelegate {
```
Comment (matcool) [inline bindings/2.200/GeometryDash.bro:7783]: this is duplicate again, literally right below ur changes

File: bindings/2.200/GeometryDash.bro [hunk 5]
```diff
@@ -7740,6 +7752,78 @@ class GJEffectManager : cocos2d::CCNode {
...
+}
+
+[[link(android)]]
+class GJAccountSettingsDelegate {
+	virtual TodoReturn updateSettingsFinished();
+	virtual TodoReturn updateSettingsFailed();
+}
+
+[[link(android)]]
+class GJCommentListLayer : cocos2d::CCLayerColor {
```
Comment (matcool) [inline bindings/2.200/GeometryDash.bro:7818]: duplicate again, all of the classes u added were already in the broma ...

**Review by matcool — CHANGES_REQUESTED** [review]

File: bindings/2.200/GeometryDash.bro [hunk 6]
```diff
@@ -7740,7 +7748,6 @@ class GJEffectManager : cocos2d::CCNode {
 	virtual bool init() = win 0x1d6f50;
 }
 
-[[link(android)]]
```
Comment (matcool) [inline bindings/2.200/GeometryDash.bro:7743]: why

File: bindings/2.200/GeometryDash.bro [hunk 7]
```diff
@@ -8646,27 +8653,26 @@ class OptionsLayer : GJDropDownLayer, FLAlertLayerProtocol {
...
+	void onHelp(cocos2d::CCObject* sender) = win 0x2ac640;
+	void onRate(cocos2d::CCObject* sender) = win 0x2ac600;
+	void onVideo(cocos2d::CCObject* sender) = win 0x2ac560;
+	void onAccount(cocos2d::CCObject* sender) = win 0x2ac5c0;
+	void onOptions(cocos2d::CCObject* sender) = win 0x2ac1b0;
+	void onSupport(cocos2d::CCObject* sender) = win 0x2ac580;
+
+	TodoReturn tryEnableRecord(); // mobile platforms only
+	void sfxSliderChanged(cocos2d::CCObject*) = win 0x2ac0b0;
+	TodoReturn createToggleButton(gd::string, cocos2d::SEL_MenuHandler, bool, cocos2d::CCMenu*, cocos2d::CCPoint); // idk lol
```
Comment (matcool) [inline bindings/2.200/GeometryDash.bro:8670]: you dont need to add "idk" to functions  you dont know

File: bindings/2.200/GeometryDash.bro [hunk 8]
```diff
@@ -8646,27 +8653,26 @@ class OptionsLayer : GJDropDownLayer, FLAlertLayerProtocol {
...
-
-	TodoReturn tryEnableRecord();
-	TodoReturn sfxSliderChanged(cocos2d::CCObject*);
-	TodoReturn createToggleButton(gd::string, cocos2d::SEL_MenuHandler, bool, cocos2d::CCMenu*, cocos2d::CCPoint);
-	TodoReturn musicSliderChanged(cocos2d::CCObject*);
-	TodoReturn exitLayer();
-
-	virtual void customSetup() = win 0x2AB510;
-	virtual TodoReturn layerHidden();
-	virtual TodoReturn FLAlert_Clicked(FLAlertLayer*, bool);
```
Comment (matcool) [inline bindings/2.200/GeometryDash.bro:8669]: why did you remove this

File: bindings/2.200/GeometryDash.bro (hunk 6 above)
Comment (Kolyah35) [reply bindings/2.200/GeometryDash.bro:7743]: ¯⁠\⁠_⁠(⁠ツ⁠)⁠_⁠/⁠¯
Maybe github merging again

File: bindings/2.200/GeometryDash.bro (hunk 8 above)
Comment (Kolyah35) [reply bindings/2.200/GeometryDash.bro:8669]: Because its unused empty function

**Comment by Kolyah35** [comment]
thanks
